### ISSUE TITLE
401 unauthorized when creating a tweet with the example code

### ISSUE THREAD

**Issue by VerburgtJimmy** [michimani/gotwi#254]
I am trying to create a tweet with this code:
`in := &gotwi.NewClientInput{
		AuthenticationMethod: gotwi.AuthenMethodOAuth1UserContext,
		OAuthToken:           "My twitter api key",
		OAuthTokenSecret:     "my twitter api key secret",
	}

	c, err := gotwi.NewClient(in)
	if err != nil {
		fmt.Println(err)
		return
	}

	p := &types.CreateInput{
		Text: gotwi.String("test"),
		Reply: &types.CreateInputReply{
			InReplyToTweetID: id,
		},
	}

	res, err := managetweet.Create(context.Background(), c, p)
	if err != nil {
		fmt.Println(err.Error())
		return
	}

	fmt.Printf("[%s] %s\n", gotwi.StringValue(res.Data.ID), gotwi.StringValue(res.Data.Text))`

But I get a 401 unauthorized. 
I have also registered a app on developer.twitter 

Does anyone have the same issue or had this issue before?
Thanks

**Comment by michimani** [comment]
@VerburgtJimmy   
Thank you for reporting the issue.

Would it be possible for you to run the following code to see the details of the error?

```go
res, err := managetweet.Create(context.Background(), c, p)
if err != nil {
	fmt.Println(err)

	// more error information
	ge := err.(*gotwi.GotwiError)
	if ge.OnAPI {
		fmt.Println(ge.Title)
		fmt.Println(ge.Detail)
		fmt.Println(ge.Type)
		fmt.Println(ge.Status)
		fmt.Println(ge.StatusCode)

		for _, ae := range ge.APIErrors {
			fmt.Println(ae.Message)
			fmt.Println(ae.Label)
			fmt.Println(ae.Parameters)
			fmt.Println(ae.Code)
			fmt.Println(ae.Code.Detail())
		}
	}
}
```

**Comment by Conor-Fleming** [comment]
@VerburgtJimmy chiming in late here... hopefully you got it sorted... but incase you havent, I experienced similar and it ended up being an issue with authorization keys. 

I found that if you made any changes to the permissions of your app in the twitter dev portal you will need to regenerate the keys for those new settings. Try regenerating the keys and seeing if that helps out...In the off chance you are still struggling with this.
